run-devel.sh

```diff
@@ -10,7 +10,7 @@ function usage() {
     exit 0
 }
 
-ROS_DISTRO="noetic"
+. settings.sh  # set initial values
```

Comment:
Personally, I do not like this strategy. Please check my other comment about this.